src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile2.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+
+// Given_StorageFile2 , as Given_StorageFile is used in PR#2407
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given_StorageFile2
+	{
+		
+		String _filename;
+		
+		[TestInitialize]
+		public void Init()
+		{
+			_filename = DateTime.Now.ToString("yyyyMMddHHmmss") + ".txt";
+		}
+
+		[TestCleanup]
+		public void Cleanup()
+		{
+		}
+
+		[TestMethod]
+		public async void When_DateCreated()
+		{
+			
+			var _folder = Windows.Storage.ApplicationData.Current.LocalFolder;
+			Assert.IsNotNull(_folder, "cannot get LocalFolder - error outside tested method");
+
+			Windows.Storage.StorageFile _file = null;
+			
+      		DateTimeOffset _dateBefore = DateTimeOffset.Now;
+      
+			try
+			{
+				_file = await _folder.CreateFileAsync( _filename, Windows.Storage.CreationCollisionOption.FailIfExists);
+				Assert.IsNotNull(_file, "cannot create file - error outside tested method");
+			}
+			catch
+			{
+			  Assert.IsNotNull(null, "CreateFile exception - error outside tested method");
+			}
+
+      		DateTimeOffset _dateAfter = DateTimeOffset.Now;
+
+			// test of DateCreated
+
+			// now, some wait - to be sure that returned date is not simply 'current date'
+			// FAT has two seconds resolution - so wait should be longer
+			await Task.Delay(5000);
```

Comment:
Don't add arbitrary delays, this slows down overall tests tests.

Reply:
this delay is required to really test DateCreated - it has to be more than twice DateCreated resolution. Test should check if DateCreated is within range, and range is defined by two points, each with 2 sec resolution.